buildspec_publish_dockerhub.yml

```diff
@@ -0,0 +1,32 @@
+version: 0.2         
+phases:
+  install:
+    runtime-versions:
+      golang: 1.12
+  pre_build:
+    commands:
+      - echo Publish the image to DockerHub
+  build:
+    commands:
+      # Pull the image that we built and pushed in the `Build` stage
+      - 'ecs-cli pull amazon/aws-for-fluent-bit:latest'
+                                                               
+      # When we pull an image from ECR, it comes with accountId, ECR registry, and region name.
+      # Tag the image only with the name.
+      # Don't forget to update your account and region in the image registry.
```

Comment:
I don't understand this comment

Reply:
Yah. Earlier we had hardcoded account id and region. I updated the codes but forgot to remove the comment. Good catch.